common/models/user.js

```diff
@@ -319,7 +319,7 @@ module.exports = function(User) {
       options.port +
       options.restApiRoot +
       userModel.http.path +
-      userModel.confirm.http.path +
+      '/confirm' +
```

Comment:
Maybe, but there isn't a way that I am aware of to get the information dynamically. Using Model.remoteMethod does not make these properties available like loopback.remoteMethod does.